assets/js/membraneWebRTC.ts

```diff
@@ -208,6 +208,31 @@ export class MembraneWebRTC {
           deserializedMediaEvent.data.id,
           deserializedMediaEvent.data.peersInRoom
         );
+
+        const turnServers = deserializedMediaEvent.data.turn_servers;
+        if (!this.rtcConfig.iceServers) {
+          this.rtcConfig.iceServers = [];
+        }
```

Comment:
This way we will always override what user passed when creating an instance of `MembraneWebRTC` class. 
Shouldn't we do something like this

Reply:
These lines sets `this.rtcConfig.iceServers` to `[]` for example, in case, when it was set ealier to `undefined` or `nil`. I didn't see any reason, to override STUN and TURN servers passed by user, even if we are not using integrated TURN

Reply:
Of course, my bad I read this incorrectly :man_facepalming: